src/runtime/manifest.ts

```diff
@@ -1241,12 +1255,16 @@ ${e.message}
     } else {
       model = entities.map(value => ({id: value.id, value}));
     }
-    await store.fromLiteral({version, model});
+
+    // TODO(lindner): fromLiteral is not declared in the StorageProviderBase/StorageStub interface
+    // tslint:disable-next-line: no-any
+    await (store as any).fromLiteral({version, model});
```

Comment:
@shans should we add fromLiteral to the public StorageStub/StorageProviderBase API?